main.go

```diff
@@ -97,6 +99,26 @@ func main() {
 		output.Hint = fmt.Sprintf("Don't forget to update the registry config:\n\n\tnpm config set @buf:registry https://%s/gen/npm/v1/\n\n", remote) +
 			fmt.Sprintf("For private modules you'll need to set the token:\n\n\tnpm config set //%s/gen/npm/v1/:_authToken {token}\n\n", remote) +
 			"See https://docs.buf.build/bsr/remote-packages/npm#private for more details."
+	case registryv1alpha1.PluginRegistryType_PLUGIN_REGISTRY_TYPE_MAVEN:
+		syntheticVersion := fmt.Sprintf(
+			"%s.%d.%s.%s",
+			// pluginResp.version does not have the leading v, but semver expects the leading v.
+			// Add it on, canonicalize it, and then remove it.
+			strings.TrimPrefix(semver.Canonical(pluginResp.version), "v"),
+			pluginResp.revision,
+			moduleResp.commitCreateAt.UTC().Format(layout),
+			moduleResp.commitName[:12],
+		)
+		output.Version = syntheticVersion
+		// remoteComponents := strings.Split(remote, ".")
+		// var groupID string
+		// for i := len(remoteComponents) - 1; i >= 0; i-- {
+		// 	groupID += remoteComponents[i] + "."
+		// }
+		// groupID += "gen"
+		// mavenName := fmt.Sprintf("%s_%s_%s_%s", moduleRef.owner, moduleRef.name, pluginRef.owner, pluginRef.name)
+		// No real one-liner for mvn/gradle, so omitting output.Command.
```

Comment:
I was shoving the commented code above into `output.Command`, but there isn't _really_ a command to both install a package and add it to your build file (and it would also be different between Maven and Gradle). Suggestions open, I can just remove it.

Reply:
Linking to docs seems reasonable. Can always use a go template or longer docstring, no strong opinion... whatever works.